### DISCUSSION TITLE
snapshot api documentation needs some beefing up

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#53549] [General]
Hello

This thread https://community.grafana.com/t/snapshot-using-http-api-does-nothing/8834/17 @marefr 

Any progress on the adding the wonderful details you provided to the documentation (https://grafana.com/docs/grafana/next/developers/http_api/snapshot/)

There are occasions where one needs to provide these reports in an automated fashion.

This 

Note: When creating a snapshot using the API, you have to provide the full dashboard payload including the snapshot data. This endpoint is designed for the Grafana UI.

is very obscure.

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
